internal/client/client_test.go

```diff
@@ -63,6 +63,13 @@ func TestExtractRelevantResourceNames(t *testing.T) {
 			exclude:   []string{"baz"},
 			expected:  []string{"bar.puorg"},
 		},
+		{
+			testName:  "two filtered by fully-qualified resource name",
+			resources: []v1.APIResource{{Name: "foo"}, {Name: "bar"}},
```

Comment:
Uagh! Can't believe I wrote such a test setup. If you have the energy, could you combine the two lists into a single `[]groupResource`? But don't feel obliged :wink:

Reply:
Changed, but not really sure if I got what you meant.